bokehjs/src/lib/models/comparisons/index.ts

```diff
@@ -0,0 +1,3 @@
+export {Comparison}   from "./comparison"
+export {CustomJSCompare} from "./customjs_compare"
+export {NanSorter} from "./nan_sorter"
```

Comment:
This `NanSorter` sticks out in the naming scheme. Maybe all should be sorters, i.e. `Sorter`, `CustomJSSorter`, `NanSorter`? In fact either way would be fine, as long as it's consistent.

Reply:
It had bothered me a bit, too. I will probably go the other direction, i.e. `NanCompare` ("comparison" seems better than "sorter" in the general sense)

Reply:
https://github.com/bokeh/bokeh/pull/13973/commits/4d03eeca5ade80337636ad5f3a8fba5a9b44b8ab